cprover_bindings/src/goto_program/expr.rs

```diff
@@ -498,8 +498,8 @@ impl Expr {
     }
 
     /// Casts value to new_typ, only when the current type of value
-    /// is equivalent to new_typ on the given machine (e.g. i32 -> c_int)
-    pub fn cast_to_machine_equivalent_type(self, new_typ: &Type, mm: &MachineModel) -> Expr {
+    /// is equivalent to new_typ on the given target (e.g. i32 -> c_int)
+    pub fn cast_to_target_equivalent_type(self, new_typ: &Type, mm: &MachineModel) -> Expr {
```

Comment:
Just to double check. There is no real change here, right? I'm guessing you are just trying to keep the same name schema as the rust compiler which makes sense to me.

Reply:
Confirmed: this was just an extension of what upstream did.